### ISSUE TITLE
Unable to Install Accessibility Agents on Windows 11: NativeCommandError from PowerShell

### ISSUE THREAD

**Issue by trodick** [Community-Access/accessibility-agents#153]
When I try to install the accessibility agents on Windows 11 25H2 (build 10.26200.8655), using the native PowerShell you can access from the Windows+X menu, I get the following output:

```text
cd c:\
c:\> irm https://raw.githubusercontent.com/Community-Access/accessibility-agents/main/install.ps1 | iex              
                                                                                                                        
  Downloading A11y Agent Team...                                                                                        
git : warning: ignoring invalid symlink type '(builtin)true' for 'claude-code-plugin/CHANGELOG.md'                      
At line:57 char:5                                                                                                       
+     git clone --quiet https://github.com/Community-Access/accessibili ...                                             
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                 
    + CategoryInfo          : NotSpecified: (warning: ignori...n/CHANGELOG.md':String) [], RemoteException              
    + FullyQualifiedErrorId : NativeCommandError
```

It appears that something isn't set up right on the remote side, but I'm not proficient enough with Git and the underlying infra to try and troubleshoot this further. Thanks!

**Comment by trodick** [comment]
Bumping this back to the top as it still hasn't been fixed in over a month. Thanks!
